### PR TITLE
NorthSoft 

### PR DESCRIPTION
**Pull request by HyperGaming99**
My Minecraft Server

### CODE REVIEW

File: minecraft_servers/northsoft/background.png
Comment (maaaathis) [inline minecraft_servers/northsoft/background.png:1]: Compress this file

File: minecraft_servers/northsoft/background@2x.png
Comment (maaaathis) [inline minecraft_servers/northsoft/background@2x.png:1]: Compress this file

File: minecraft_servers/northsoft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "server_name": "northsoft",
+    "nice_name": "NorthSoft Netzwerk",
+    "direct_ip": "northsoft.eu",
+    "supported_languages": [
+       "de"
+    ],
+    "social": {
+      "discord": "https://dsc.gg/northsoft",
```
Comment (maaaathis) [inline minecraft_servers/northsoft/manifest.json:9]: Custom links are reserved for partners only; use a normal discord.gg/[YOUR-INVITE] link

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by HyperGaming99** [comment]
Updatet

**Review by DoJapHD — APPROVED** [review]

**Comment by HyperGaming99** [comment]
Thanks vor Merge

**Comment by HyperGaming99** [comment]
Can you remove Northsoft from LabyMod

**Pull request merged**